### PR TITLE
docs: add tsdoc documentation output

### PR DESCRIPTION
**Pull request by lance**
This is replacing the generated JSDoc here https://cloudevents.github.io/sdk-javascript/.

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

**Comment by lance** [comment]
@cloudevents/sdk-javascript-maintainers ping - I will land this today unless there are objections.

File: docs/assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,2321 @@
+/*! normalize.css v1.1.3 | MIT License | git.io/normalize */
```
Comment (grant) [inline docs/assets/css/main.css:1]: This file is copied from `normalize.css`. It should live in that repo, not copied to this repo.

**Comment by lance** [comment]
I will resubmit for `github-pages` branch

**Comment by lance** [comment]
> Pinging is fine, but I don't think we should assume we can land changes without approval. We need some time outside of the weekend.

This is exactly what this issue was meant to discuss... https://github.com/cloudevents/sdk-javascript/issues/86

**Pull request closed without merging**